Firmware/MotorControl/utils.h

```diff
@@ -13,11 +75,11 @@
 // as per the magnitude invariant clarke transform
 // The magnitude of the alpha-beta vector may not be larger than sqrt(3)/2
 // Returns 0 on success, and -1 if the input was out of range
-int SVM(float alpha, float beta, float* tA, float* tB, float* tC);
+int SVM(float alpha, float beta, float *tA, float *tB, float *tC);
```

Comment:
why did you change this?

Reply:
I auto-formatted the document to clean up the defines and I guess it didn't like where the *'s were.